src/SDammann.WebApi.Versioning/Properties/AssemblyInfo.cs

```diff
@@ -11,6 +11,6 @@
 [assembly: AssemblyCulture("")]
 [assembly: ComVisible(false)]
 [assembly: Guid("bd7cdad1-3775-49f2-85ed-0a0571c74f12")]
-[assembly: AssemblyVersion("2.7.0.0")]
-[assembly: AssemblyInformationalVersion("2.7.0.0")]
-[assembly: AssemblyFileVersion("2.7.0.0")]
+[assembly: AssemblyVersion("2.9.1.1")]
+[assembly: AssemblyInformationalVersion("2.9.1.1")]
```

Comment:
Since 2.7 is the current version I won't merge this and just keep the existing increment